index.html

```diff
@@ -0,0 +1,327 @@
+                          or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="skill-portfolio">
+                          <span class="badge"
+                            >HTML</span
+                          >
+                          <span class="badge"
+                            >Bootstrap</span
+                          >
+                          <span class="badge"
+                            >Ruby</span>
+                        </div>
+                        <div class="project-see">
+                          <h1 class="see-project">see project </h1>
+                        </div>
+                       </div>
+                    </div>
+                    </div>
+                    </div>
+                    <div class="row-portfolio">
+                      <div class="card-portfolio">
+                        <div class="card-6">
+                          <div class="card-profession">
+                            <h3 class="heading">Professional Art Printing Data</h3>
+                            <p class="card-text-portfolio">
+                              A daily selection of privately personalized reads; no accounts
+                              or sign-ups required. has been the industry's standard
+                            </p>
+                            <div class="skill-portfolio">
+                              <span class="badge bg-secondary text-dark text-white"
+                                >HTML</span
+                              >
+                              <span class="badge bg-secondary text-dark text-white"
+                                >Bootstrap</span
+                              >
+                              <span class="badge bg-secondary text-dark text-white"
+                                >Ruby</span>
+                            </div>
+                            <div class="project-see">
+                              <h1 class="see-project">see project</h1>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+               </div>  
+        </section>    
+        <section id="about" class="container">
+        <div class="row-about">
+          <div class="about">
+            <h2 class="about-heading">About me</h2>
+            <p class="about-text">
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <button type="button" class="btn">
+              Get My Resume
+            </button>
+          </div>
+          <div class="about">
+            <img src="images/group3.png" class="img-fluid2" alt="About me section decoration"/>
+          </div>
```

Comment:
[OPTIONAL]

- [ ] This is a background image, which means you should add it in CSS as a `background-image` instead of adding it as an image here, I suggest you change that, please. 😉